tests/realm.cpp

```diff
@@ -1580,7 +1580,7 @@ TEST_CASE("Statistics on Realms") {
     });
 
     SECTION("compute_size") {
-        auto s = r->compute_size();
+        auto s = r->read_group().compute_aggregated_byte_size();
```

Comment:
You probably want to add `SizeAggregateControl::size_of_state`. Otherwise you will always get 0 in return.